packages/sources/src/components/highlight-box/highlight-box.scss

```diff
@@ -37,6 +37,16 @@ $dso-highlight-box-box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
     *:first-child {
       margin-top: 0;
     }
+    
+    img:first-child {
```

Comment:
Ik krijg hier 'n beetje banner-achtige vibes, wat ik een leuke uitbreiding op het Highlight Box component vind, maar moeten we dat dan niet explicieteren met `img.dso-highlight-box-banner` ?

Deze selector is namelijk erg invasief: Toevallig ben ik er vandaag achtergekomen dat regel 37 gewoon een harde bug is, en met deze selector doen we hetzelfde: "Elke img:eerste-kindje moet je deze styling geven". Maar dat img:eerste-kindje kan ook dieper staan en die wordt dan ook geraakt door deze selector.

Reply:
class dso-highlight-box-banner toegevoegd op de selector image in highlightbox